detector/calorimeter/README.md

```diff
@@ -34,13 +34,16 @@ This sub-detector makes calorimeter barrel. It is used in ALLEGRO detector conce
 
 ### o1_v01
 Original version taken from [FCCDetectors](https://github.com/HEP-FCC/FCCDetectors/blob/70a989a6fc333610e3b1b979c3596da9c41543d8/Detector/DetFCChhHCalTile/src/HCalBarrel_geo.cpp). 
+In August 2024, added extension (LayeredCalorimeterData) to store radial layer radii and dimensions;  made small updates of the code to improve readibility, while the functionality remains the same.
 
 ## HCalThreePartsEndcap
 This sub-detector makes calorimeter endcaps. Each endcap is made up by three cylindrical pieces with different thickness and inner radius, but same outer radius. It is used in ALLEGRO detector concept.
 
 ### o1_v01
 Original version taken from [FCCDetectors](https://github.com/HEP-FCC/FCCDetectors/blob/70a989a6fc333610e3b1b979c3596da9c41543d8/Detector/DetFCCeeHCalTile/src/HCalThreePartsEndcap_geo.cpp#L4). 
 
+### o1_v02
+Changes wrt o1_v01: Added extension (LayeredCalorimeterData) to store radial layer radii and dimensions. To make this work, the whole code had to be rewritten, but its functionality remains the same. 
```

Comment:
In the current status of the PR, v01 also has the extension. By reading both comments from v01 and v02, it is not clear what is the difference between them... It feels like they would end-up being exactly the same file (especially "To make this work, the whole code had to be rewritten" which then implies that v01 also had to be completely rewritten since it also has the extension). Do we actually need to modify v01? If yes, do we need v02?

Reply:
Sorry for the confusion @BrieucF . it should be only HCalThreePartsEndcap_o1_v02 that is changed and includes the extension now, while v01 is untouched and without the extension. Looking back now, what might be confusing is that I added extension to HCalTileBarrel_o1_v01_geo.cpp and probably it would be cleaner to have v02 as the version with extensions for both barrel and endcap. what do you think?

Reply:
Yes, let's apply the changes in v02 and keep v01 untouched (and update the README accordingly)

Reply:
Yeah @BrieucF  I agree that it'll be better this way and it's done in the last commit